features/steps/steps.py

```diff
@@ -217,6 +310,17 @@ def step_impl(context, related, relating, other_entity, condition):
             for inst in context.model.by_type(related):
                 for rel in getattr(inst, 'Decomposes', []):
                     if not rel.RelatingObject.is_a(relating):
-                        errors.append(instance_structure_error(inst, rel.RelatingObject))
+                        errors.append(instance_structure_error(inst, [rel.RelatingObject], 'assigned to'))
 
     handle_errors(context, errors)
+
+@then ('The value of attribute {attribute} should be of type {expected_entity_type}')
+def stemp_impl(context, attribute, expected_entity_type):
+
+    def _():
+        for inst in context.instances:
+            related_entity = getattr(inst, attribute, [])
+            if not related_entity.is_a(expected_entity_type):
```

Comment:
As mentioned earlier, we need to get `is_a(s)` and `is_a() == s` consistent,

Reply:
All set to is_a(s) in https://github.com/buildingSMART/ifc-gherkin-rules/commit/0c8fdcad7ba2cb8627485193daa2643b93e770a3